client/ethereum.go

```diff
@@ -525,7 +526,16 @@ func (e *EthereumClient) SendTransaction(
 	if err != nil {
 		return common.Hash{}, err
 	}
+
+	// https://docs.klaytn.com/klaytn/design/transaction-fees#unit-price
```

Comment:
Create a `KlaytnClient` that inherits the ETH client and override these methods so it doesn't dirty up the main client with string matching

Reply:
I was considering this, but since it's such a tiny difference, figured it was maybe not worth. But ya, less string matching == better.